packages/web-components/src/card/card.ts

```diff
@@ -0,0 +1,207 @@
+import { CardAppearance, CardControlSize, CardOrientation } from './card.options.js';
+
+/**
+ * Card configuration options
+ * @public
+ */
+export type CardOptions = StartEndOptions<Card>;
+
+/**
+ * @class Card component
+ *
+ * @remarks
+ * This class extends the FASTCard. a flexible content container
+ */
+export class Card extends FASTCard {
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this.setInteractive();
+    this.updateComputedStylesheet();
+  }
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+  }
+
+  /**
+   * Stores the computed stylesheet for the card
+   */
+  private computedStylesheet?: ElementStyles;
+
+  /**
+   * @property orientation;
+   * @default filled
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public appearance?: CardAppearance;
+
+  /**
+   * @property orientation;
+   * @default vertical
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public orientation?: CardOrientation;
+
+  /**
+   * @property controlSize
+   * @default medium
+   * @remarks
+   * Determines the size of the card
+   */
+  @attr({ attribute: 'control-size' })
+  public controlSize?: CardControlSize;
+
+  /**
+   * @property interactive
+   * @default false
+   * @remarks
+   * Determines whether card is interactable
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public interactive: boolean = false;
+
+  /**
+   * @property disabled
+   * @default false
+   * @remarks
+   * Determines disabled state of card
+   */
+  @attr({ mode: 'boolean' })
+  public disabled: boolean = false;
+
+  /**
+   * @property selected
+   * @default false
+   * @remarks
+   * Determines selected state of card
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public selected: boolean = false;
+
+  /**
+   * @method sizeChanged
+   * @remarks
+   * Updates the computed stylesheet when the size of the card changes
+   */
+  public sizeChanged(prev: boolean | undefined, next: boolean): void {
+    this.updateComputedStylesheet();
+  }
+
+  /**
+   * @method selectedChanged
+   * @remarks
+   * Emits an event when the selected state of the card changes
+   */
+  public selectedChanged = (): void => {
+    this.$emit('onSelectionChanged', this.selected);
+  };
+
+  /**
+   * @method interactiveChanged
+   * @remarks
+   * Updates the focusable state of the card when the interactive state changes
+   */
+  public interactiveChanged = (): void => {
+    this.setInteractive();
+  };
+
+  /**
+   * @method setFocusable
+   * @remarks
+   * Sets the tabindex attribute based on the interactive state of the card
+   * @internal
+   */
+  public setInteractive(): void {
+    if (this.interactive) {
+      this.setAttribute('tabindex', '0');
+      this.setAttribute('aria-selected', this.selected ? 'true' : 'false');
+    } else {
+      this.removeAttribute('tabindex');
+      this.removeAttribute('aria-selected');
+    }
+  }
```

Comment:
Why not set this in the template based on whether or not the card is interactive?